src/components/DropdownSearch/index.js

```diff
@@ -107,6 +109,21 @@ function DropdownSearch({
   };
 
   const icon = nav && !suggestions?.length ? NavSearchIcon : SearchIcon;
+  const searchIconButton = (
```

Comment:
Rather than this component checking for which type of icon to show, label to show as placeholder, etc., can it just display what it has been given? That way, the logic will be in the components using this component. For example, here will pass in the correct icon depending on if it has suggestions or not.